src/sql/workbench/contrib/dashboard/browser/core/dashboardToolbarHomeAction.contribution.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IExtensionPointUser, ExtensionsRegistry } from 'vs/workbench/services/extensions/common/extensionsRegistry';
+import { IJSONSchema } from 'vs/base/common/jsonSchema';
+import { localize } from 'vs/nls';
+import { registerToolbarHomeAction } from 'sql/workbench/contrib/dashboard/browser/dashboardRegistry';
+
+export interface IDashboardToolbarHomeActionContrib {
```

Comment:
I'm not sure I'd recommend special-casing the home page for toolbar contributions.  Ideally we'd have a more generic contribution that would allowing adding toolbar buttons to any page, and there would be some sort of target attribute that would specify which toolbar should have the action.

Maybe I missed something, but I thought that "home" just means "default" and that would be a different page depending on the type of dashboard being shown.